example: leafmap

akshayka opened the pull request:
Use leafmap.foliumap, since leafmap.Map is not supported

mscolnick commented:
I fixed this in leafmap if you upgrade, maybe try upgrading to the latest and it should automatically pick folium

akshayka commented:
> I fixed this in leafmap if you upgrade, maybe try upgrading to the latest and it should automatically pick folium

Oh thanks. Updated the dependency in the metadata.
